app/mailers/user_mailer.rb

```diff
@@ -10,25 +10,28 @@ def donation_available(recipient_ids, donation)
     end
   end
 
-  def recipient_match(recipients)
-    if recipients.length > 0
-      mail(to: recipients, subject: "Your donation has found a recipient!")
+  def coordinator_email(coordinator_ids, donation)
+    @donation = donation
+    @profile = DonorProfile.find_by donor_id: donation.donor_id
+    if coordinator_ids.length > 0
+      coordinator_ids.each do |c|
+        @coordinator = Coordinator.find(c)
+        mail(to: @coordinator.email, subject: "(Coordinator) Donation Posted!")
+      end
     end
   end
 
-  def coordinator_email(recipients, donation)
-    if recipients.length > 0
-      @donation = donation
-      @profile = DonorProfile.find_by donor_id: donation.donor_id
-      mail(to: recipients, subject: "(Coordinator) Donation Posted!")
-    end
+  def coordinator_matched_donor(donation, donor_id, recipient_id)
```

Comment:
Unused method argument - `recipient_id`. If it's necessary, use `_` or `_recipient_id` as an argument name to indicate that it won't be used.